app/_includes/md/gateway/setup.md

```diff
@@ -186,7 +186,7 @@ If you're running {{site.base_gateway}} with a database (either in traditional
 or hybrid mode), you can enable {{site.base_gateway}}'s graphical user interface
 (GUI), Kong Manager.
 
-See the [Kong Manager setup guide](/gateway/{{page.kong_version}}/kong-manager/enable/) for more information.
+See the [Kong Manager setup guide](/gateway/{{page.kong_version}}/kong-manager/enable/){% if_version gte:3.4.x %} or the [Kong Manager OSS guide](/gateway/{{page.kong_version}}/kong-manager-oss){% endif_version %} for more information.
```

Comment:
❗ For reviewers: On line 181, I wanted to make the badge conditional because it only applies to 3.3.x or earlier. But any if_version formatting I added made the formatting all weird. Any ideas?

Reply:
i think you have to use conditionals for the whole header, eg:

```
{% if_version lte:3.3.x %}

### Enable Kong Manager
{:.badge .free}

{% endif_version %}
```